package.json

```diff
@@ -81,7 +83,7 @@
       "<rootDir>/packages/core/"
     ],
     "testMatch": [
-      "<rootDir>/packages/core/src/__tests__/**/?(*.)(spec|test).ts(x|)"
+      "<rootDir>/packages/core/src/**/__tests__/?(*.)(spec|test).ts(x|)"
```

Comment:
@mresposito Let's maintain the `.spec` naming convention for test files. We can remove `|test` from the glob here for consistency.

Reply:
sounds good